### DISCUSSION TITLE
File discovery

### DISCUSSION THREAD

**Discussion by yeashin** [karimra/gnmic#635] [Q&A]
I'm looking for an example of configure loader within main configuration. I've tried to add loader section after targets. However, gNMIc can not find the file path. Then, I've tried to use --targets-file flags, it can find the file from the same path.  Thank you.

**Comment by karimra** [comment]
Have you checked these examples?
https://gnmic.kmrd.dev/user_guide/target_discovery/file_discovery/#examples

**Reply by yeashin** [reply]
Thank you for your quick reply. I find my mistake :)
